src/main/features/ccj/views/confirmation.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "layout.njk" %}
+{% from "includes/macros/form.njk" import submitButton, errorSummary %}
+{% from "includes/macros/support.njk" import supportWidget %}
+{% set title = t('CCJ requested') %}
+
+{% block content %}
+
+  <div class="grid-row">
```

Comment:
for whoever picks this up, this needs adding after the grid-row: 
```
<div class="column-two-thirds">
```

Reply:
do you mean:

```
  <div class="grid-row">
    <div class="column-two-thirds">
```

Reply:
yes